packages/ast-spec/tests/PunctuatorTokenToText.test.ts

```diff
@@ -0,0 +1,11 @@
+import type { PunctuationSyntaxKind } from 'typescript';
+
+import type { PunctuatorTokenToText } from '../src';
+
+// @ts-expect-error: purposely unused
+type _Test = {
+  readonly [T in PunctuationSyntaxKind]: PunctuatorTokenToText[T];
+  // If there are any PunctuationSyntaxKind members that don't have a
```

Comment:
Comment needed to be added after `readonly` line because of https://github.com/prettier/prettier/issues/11098